privacyidea/lib/resolvers/SCIMIdResolver.py

```diff
@@ -69,19 +69,37 @@ def checkPass(self, uid, password):
         # TODO: Implement password checking with SCIM
         return False
 
+    # def getUserInfo(self, userid):
+    #     """
+    #     returns the user information for a given uid.
+    #     """
+    #     ret = {}
+    #     # The SCIM ID is always /Users/ID
+    #     # Alas, we can not map the ID to any other attribute
+    #     res = self._get_user(self.resource_server,
+    #                          self.access_token,
+    #                          userid)
+    #     user = res
+    #     ret = self._fill_user_schema_1_0(user)
+
+    #     return ret
     def getUserInfo(self, userid):
         """
         returns the user information for a given uid.
         """
         ret = {}
         # The SCIM ID is always /Users/ID
         # Alas, we can not map the ID to any other attribute
-        res = self._get_user(self.resource_server,
-                             self.access_token,
-                             userid)
-        user = res
-        ret = self._fill_user_schema_1_0(user)
-
+        params = {'filter':'userName eq "{0}"'.format(userid)}
+        res = self._search_users(self.resource_server,
+                                 self.access_token,
+                                 self.verify_tls,
+                                 params)
+        if len(res.get('resources', [])) > 1:
+            raise Exception('Could not find user: {0}'.format(userid))
+        if res.get('resources'):
+            for entry in res.get('resources'):
+                ret = self._fill_user_schema_1_0(entry, self.mapping)
```

Comment:
This pull request is not working and not complete.
E.g. in this case the parameters of `_fill_user_schema_1_0` is wrong, since the method has no parameter for `self.mapping`!.

I assume, that you are working on a very different base branch!
So you probably either need to add more changes or rebase your branch.

Also, it would be great if you would open an issue and refer to this issue so that we know, what this pull request is ment to fix.